lib/container/rabbitmq_container.ex

```diff
@@ -0,0 +1,259 @@
+defmodule Testcontainers.RabbitMQContainer do
+  @moduledoc """
+  Provides functionality for creating and managing RabbitMQ container configurations.
+  """
+  alias Testcontainers.ContainerBuilder
+  alias Testcontainers.Container
+  alias Testcontainers.CommandWaitStrategy
+  alias Testcontainers.RabbitMQContainer
+
+  @default_image "rabbitmq"
+  @default_tag "3-alpine"
+  @default_image_with_tag "#{@default_image}:#{@default_tag}"
+  @default_port 5672
+  @default_username "guest"
+  @default_password "guest"
+  @default_virtual_host "/"
+  @default_command [
+    "sh",
+    "-c",
+    "chmod 400 /var/lib/rabbitmq/.erlang.cookie; rabbitmq-server"
```

Comment:
hmm this erlang cookie. Whats this?

Reply:
This is included in order for allowing access to CLI tools to communicate with the RabbitMQ node, which is used in the command wait strategy, with `rabbitmq-diagnostics check_running`.  I have updated the module docs  to include this. (Relevant docs: https://www.rabbitmq.com/docs/cli#erlang-cookie)